contact.html

```diff
@@ -0,0 +1,74 @@
+
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="utf-8" />
+<title>My first coding project</title>
+
+<link href="style.css" rel="stylesheet" type="text/css" media="screen" />
+
+<!-- <script type="text/javascript" src="http://code.jquery.com/jquery-1.5.1.min.js"></script>
+<script type="text/javascript" src="js/scripts.js"></script> -->
+
+</head>
+
+<body>
+	<div id="container">
+		<p id="logo"><a href="index.html"><img class="oval" src="images/klara.jpg" alt="Klara logo" /></a></p>
+		
+		<ul id="nav">
+			<li><a href="index.html">Home</a></li>
+			<li><a href="#">About</a></li>
+			<li><a href="#">Portfolio</a></li>
+			<li><a href="contact.html">Contact</a></li>
+		</ul>
+	
+		<div id="header" class="page">
+			<h1>Get in touch</h1>
+		</div>
+	
+		<div id="content">
+			<p>Give it a try, we hear it's all the rage these days!</p>
+			
+			<div id="contact">
+				<form action="http://formspree.io/wondercodersdk@gmail.com" method="post">

```

Comment:
can you please change the action, otherwise the email will get spammed?

Reply:
Yep I will, when I looked at the code didnt notice that part :dancer: